### ISSUE TITLE
[Bug] Accessibility - Tooltip cannot be dismissed by keyboard

### ISSUE THREAD

**Issue by meganthecoder** [apache/echarts#21423]
### Version

6.0.0

### Link to Minimal Reproduction

https://codepen.io/meganthomas/pen/gbryOGo

### Steps to Reproduce

1. Hover your mouse over the Search Engine (light blue) data point for Mon. The tooltip should appear.
2. Press the escape (ESC) key on your keyboard.
3. Notice how the tooltip is still visible.

### Current Behavior

The tooltip cannot be closed by pressing the escape key. This violates WCAG Success Criteria 1.4.13: Content on Hover or Focus. According to this accessibility criteria, there must be a way to dismiss the tooltip without moving the mouse. This is most commonly done with the escape key.

More info: https://www.w3.org/WAI/WCAG22/Understanding/content-on-hover-or-focus.html

### Expected Behavior

When the tooltip is visible, pressing the escape key should close it.

### Environment

```markdown
- OS: macOS Sequoia
- Browser: Chrome 142.0.7444.177
- Framework: N/A
```

**Comment by helgasoft** [comment]
In a previous discussion (#21022), I provided a Demo for keyboard access to charts. The chart needs **focus** to accept keyboard events. Hiding a tooltip is then simple with ```dispatchAction({ type:'hideTip'...})```.
